sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -3636,20 +3636,24 @@ SIREPO.app.directive('moderationRequest', function(appState, errorService, panel
               <label for="requestAccessExplanation">Please describe your reason for requesting access:</label>
               <textarea data-ng-show="!submitted" data-ng-model="data.reason" id="requestAccessExplanation" class="form-control" rows="4" cols="50" required></textarea>
             </div>
-            <button data-ng-show="!submitted" type="submit" class="btn btn-primary" data-ng-click="submitRequest()">Submit</button>
+            <button data-ng-disabled="disableSubmit" data-ng-show="!submitted" type="submit" class="btn btn-primary" data-ng-click="submitRequest()">Submit</button>
```

Comment:
Can you use disableAfterClick? https://github.com/radiasoft/sirepo/blob/bf88fb821bb4190771590fc2b6b12fc44070bbc2/sirepo/package_data/static/js/sirepo-components.js#L519

Reply:
What we want is to enable the button only when the user enters something, similar to the way "Save changes" works.